common/handy/maps.go

```diff
@@ -0,0 +1,45 @@
+package handy
+
+type Map[K comparable, V any] map[K]V
```

Comment:
Let's use something battle tested like https://github.com/deckarep/golang-set? Could be pretty handy later on

Reply:
It doesn't use generics but rather `interface{}`. Type checking is much needed.
Lets keep this implementation for now. Later we could replace it with alternative.